pypeit/par/pypeitpar.py

```diff
@@ -1652,213 +1659,7 @@ def validate(self):
-        dtypes['pixelmap'] = str
-        descr['pixelmap'] = 'If desired, a fits file can be specified (of the appropriate form)' \
-                            'to specify the locations of the pixels on the detector (in ' \
-                            'physical space).  TODO: Where is "appropriate form" specified?'
-
-        defaults['pixelwidth'] = 2.5
-        dtypes['pixelwidth'] = [int, float]
-        descr['pixelwidth'] = 'The size of the extracted pixels (as an scaled number of Arc ' \
-                              'FWHM), -1 will not resample'
-
-        defaults['reuse'] = False
-        dtypes['reuse'] = bool
-        descr['reuse'] = 'If the extraction has previously been performed and saved, load the ' \
-                         'previous result'
-
-        defaults['profile'] = 'gaussian'
-        options['profile'] = ExtractObjectsPar.valid_profiles()
-        dtypes['profile'] = str
-        descr['profile'] = 'Fitting function used to extract science data, only if the ' \
-                           'extraction is 2D.  NOTE: options with suffix \'func\' fits a ' \
-                           'function to the pixels whereas those without this suffix take into ' \
-                           'account the integration of the function over the pixel (and is ' \
-                           'closer to truth).   ' \
-                           'Options are: {0}'.format(', '.join(options['profile']))
-
-        dtypes['maxnumber'] = int
-        descr['maxnumber'] = 'Maximum number of objects to extract in a science frame.  Use ' \
-                             'None for no limit.'
-
-        dtypes['manual'] = list
-        descr['manual'] = 'List of manual extraction parameter sets'
-
-        # Instantiate the parameter set
-        super(ExtractObjectsPar, self).__init__(list(pars.keys()),
-                                                values=list(pars.values()),
-                                                defaults=list(defaults.values()),
-                                                options=list(options.values()),
-                                                dtypes=list(dtypes.values()),
-                                                descr=list(descr.values()))
-        self.validate()
-
-    @classmethod
-    def from_dict(cls, cfg):
-        k = cfg.keys()
-        parkeys = [ 'pixelmap', 'pixelwidth', 'reuse', 'profile', 'maxnumber' ]
-        kwargs = {}
-        for pk in parkeys:
-            kwargs[pk] = cfg[pk] if pk in k else None
-        kwargs['manual'] = util.get_parset_list(cfg, 'manual', ManualExtractionPar)
-        return cls(**kwargs)
-
-    @staticmethod
-    def valid_profiles():
-        """
-        Return the list of valid functions to use for object tracing.
-        """
-        return [ 'gaussian', 'gaussfunc', 'moffat', 'moffatfunc' ]
-
-    def validate(self):
-        pass
-
-# ToDO place holder to be updated by JFH
 class ScienceImagePar(ParSet):
```

Comment:
this is probably not the time to do this, but I should love for the Class names here to be identical to their names in the parset.  Just sayin'

Reply:
You mean you want this to be called `ReducePar` to match that these are sent to the `Reduce` class? If so, then I have more comments/questions.  But yes, I think we should add this as an issue for a later PR.

Reply:
I agree with this point. However there is something else called reduce I think, i.e. the top-level parset.

Reply:
Let's add this as a topic for the next telecon.